mne/viz/evoked.py

```diff
@@ -1822,6 +1831,22 @@ def _get_data_and_ci(evoked, scaling=1, picks=None, ci_fun=None, gfp=False):
         return data,
 
 
+def _calculate_ci_and_mean(evokeds, conditions, scaling, picks, ci_fun, gfp):
+    """Caluclate time series and CI, potentially aggregating over sensors."""
+    ci_dict, data_dict = dict(), dict()
+
+    for cond in conditions:
+        this_evokeds = evokeds[cond]
+        # this will fail if evokeds do not have the same structure
```

Comment:
we have some function to check this, right? Something like this is being checked when doing ts or tfr grand averages.

Reply:
Yes, and this is being done in this function, line 1592.